editor/src/clj/editor/gui.clj

```diff
@@ -307,13 +315,14 @@
       corners? (cornify max-angle)
       cut-off? (into (geom/rotate [0 0 max-angle] ps)))))
 
-(defn- v3->v4 [v3 default]
-  (conj (or v3 default) 1.0))
+(defn- v3->v4 [v3]
+  (conj v3 1.0))
 
 (def ^:private prop-index->prop-key
   (let [index->pb-field (protobuf/fields-by-indices Gui$NodeDesc)
         renames {:xanchor :x-anchor
-                 :yanchor :y-anchor}]
+                 :yanchor :y-anchor
+                 :size :manual-size}]
```

Comment:
`(property size ...)` was renamed to `manual-size` in the graph nodes. We use it to store manual size settings only.